sdk/src/cameras/itof-camera/mode_info.h

```diff
@@ -76,6 +77,25 @@ class ModeInfo {
     aditof::Status convertCameraMode(const std::string &modes,
                                      uint8_t &convertedMode);
 
+    /**
+     * Get the sensor width and height for the provided mode
+     * param[in] modes - the name of the mode (represented as a string)
+     * param[out] width - sensor width
+     * param[out] height - sensor height
+     * @return aditof::Status
+     */
```

Comment:
also add description for pixelFormatIndex

Reply:
Fixed.